pennylane/measurements/vn_entropy.py

```diff
@@ -61,4 +64,13 @@ def circuit_entropy(x):
     .. seealso:: :func:`pennylane.qinfo.transforms.vn_entropy` and :func:`pennylane.math.vn_entropy`
     """
     wires = Wires(wires)
-    return MeasurementProcess(VnEntropy, wires=wires, log_base=log_base)
+    return _VnEntropy(VnEntropy, wires=wires, log_base=log_base)
+
+
+class _VnEntropy(StateMeasurement):
+    """Measurement process that returns the Von Neumann entropy."""
```

Comment:
So the only measurement processes that use the `log_base` argument are `vn_entropy` and `mutual_info`. I think we should i) remove the `log_base` argument from the `__init__` method of `MeasurementProcess` and ii) add `__init__` methods for the `MutualInfo` and `VnEntropy` classes to include this argument.

This will help declutter the code in the base class.

Reply:
Nice catch! Moved `log_base` to children classes